index.js

```diff
@@ -35,7 +35,7 @@ var printIssue = function(fileName, lineNum, matchedString){
 
 var findFixmes = function(file){
   // Prepare the grep string for execution (uses BusyBox grep)
-  var grepString = "grep -inHwoE " + fixmeStrings + " " + file;
+  var grepString = "grep -inHwoE " + fixmeStrings + " \"" + file + "\"";
```

Comment:
Nice! I like your fix. I don't think it's better, but one other idea I had was something like:

```
var grepString = ["grep -inHwoE", fixmeStrings, "'" + file + "'"].join(" ");
```